roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -55,6 +55,45 @@
   when:
     - youki_enabled
 
+- name: Cri-o | read default container runtime from crio.conf
+  ansible.builtin.shell: |
+    set -o pipefail && grep "^default_runtime" /etc/crio/crio.conf | awk -F'=' '{print $2}' | xargs
+  register: existing_crio_default_runtime
+  when: ansible_facts.services['crio.service'] is defined
```

Comment:
crio.conf might not reflect the runtime state. I think we should use `crictl info` instead (which also has the advantages of being json).

(And, as a more general things, `shell` is to be avoided, prefer command and doing transformation in jinja)

Also, this problably lets you drop the ansibe_facts condition I think ?